src/app/components/event/steps/Preview.vue

```diff
@@ -0,0 +1,255 @@
+<template>
+  <div class="-mx-4 sm:-mx-6 md:-mx-8">
+    <div class="relative w-full">
+      <template v-if="props.form.previewUrls.length > 0">
+        <Carousel
+          v-model="currentSlide"
+          class="w-full"
+          @select="handleSlideChange"
+        >
+          <CarouselContent>
+            <CarouselItem
+              v-for="(url, index) in props.form.previewUrls"
+              :key="index"
+              class="w-full"
+            >
+              <div class="relative aspect-[16/9] w-full">
+                <img
+                  :src="url"
+                  class="h-full w-full object-cover"
+                  :alt="`Event image ${index + 1}`"
+                />
+              </div>
+            </CarouselItem>
+          </CarouselContent>
+        </Carousel>
+
+        <div class="absolute right-4 top-4 z-10 flex space-x-3">
+          <button
+            class="flex h-10 w-10 items-center justify-center rounded-full bg-white shadow hover:bg-gray-100"
+            :aria-label="t('share')"
+          >
+            <Share class="h-6 w-6 text-gray-700" />
+          </button>
+          <button
+            class="flex h-10 w-10 items-center justify-center rounded-full bg-white shadow hover:bg-gray-100"
+            :aria-label="t('favorite')"
+          >
+            <Heart class="h-6 w-6 text-red-500" />
+          </button>
+          <button
+            class="flex h-10 w-10 items-center justify-center rounded-full bg-white shadow hover:bg-gray-100"
+            :aria-label="t('options')"
+          >
+            <MoreHorizontal class="h-6 w-6 text-gray-700" />
+          </button>
+        </div>
+
+        <div
+          class="absolute bottom-4 right-4 z-10 rounded-full bg-black/70 px-3 py-1 text-sm text-white"
+        >
+          {{ currentSlide + 1 }}{{ t('slash') }}{{ form.previewUrls.length }}
+        </div>
+      </template>
+
+      <template v-else>
+        <div
+          class="aspect-[16/9] w-full rounded-2xl border-2 border-dashed border-gray-300 p-12"
+        >
+          <div
+            class="flex h-full flex-col items-center justify-center text-center"
+          >
+            <Image class="mb-4 h-12 w-12 text-gray-400" />
+            <p class="text-gray-500">{{ t('noImages') }}</p>
+          </div>
+        </div>
+      </template>
+    </div>
+
+    <div class="p-4 sm:p-6">
+      <div class="flex items-start gap-6">
+        <div class="flex flex-col items-center text-center">
+          <span class="text-primary text-lg font-medium">{{
+            formatDate(form.startDate, { weekday: 'long' })
+          }}</span>
```

Comment:
Let's continue to keep tags and texts on separate lines

```suggestion
          <span class="text-primary text-lg font-medium">
            {{ formatDate(form.startDate, { weekday: 'long' }) }}
          </span>
```

Reply:
Applies to `pages/event/preview/index.vue` too